postal/buildpack.go

```diff
@@ -10,9 +10,14 @@ import (
 
 // Dependency is a representation of a buildpack dependency.
 type Dependency struct {
-	// CPE is the Common Platform Enumerator for the dependency.
+	// CPE is the Common Platform Enumerator for the dependency. Used in legacy
+	// image label SBOM (GenerateBillOfMaterials).
 	CPE string `toml:"cpe"`
```

Comment:
Do we want to mark this as deprecated?

Reply:
That seems reasonable, given the standard SBOM formats only talk about CPEs plural.